values/roundtriptest/int_test.go

```diff
@@ -69,31 +80,57 @@ func TestNegativeIntDictionaryEncodeAndDecode(t *testing.T) {
 	testEncodeAndDecodeInt(t, data, meta, vals)
 }
 
-func TestPositiveIntDeltaEncodeAndDecode(t *testing.T) {
+func TestMixedIntRawSizeEncodeAndDecode(t *testing.T) {
```

Comment:
`TestMixedVarintEncodeAndDecode`.

Reply:
Changed to `TestMixedIntVarintEncodeAndDecode`.